airbyte-integrations/bases/base-normalization/build.gradle

```diff
@@ -59,7 +59,7 @@ def buildAirbyteDocker(String customConnector) {
     }
 
     def baseCommand = ['docker', 'buildx', 'build', '--load', '--platform', arch, '-f', getDockerfile(customConnector), '-t', getImageNameWithTag(customConnector), '.']
-    println("Building normalization container: " + baseCommand.join(" "))
+    // println("Building normalization container: " + baseCommand.join(" "))
```

Comment:
unrelated to this PR, but I was annoyed.  I added this log statement in https://github.com/airbytehq/airbyte/pull/26677, but I didn't realize it would log on *every* gradle run, even those that have nothing to do with normalization 😓